source/content/wordpress-launch-check.md

```diff
@@ -23,6 +23,24 @@ In short, you get a fast, repeatable report that can help detect common problems
 
 WP Launch Check is a site audit extension for WP-CLI designed for Pantheon customers. While designed initially for the Pantheon Dashboard it is intended to be fully usable outside of Pantheon.
 
+## WPScan Dependency
+
+WP Launch Check utilizes the [WPScan plugin](https://wordpress.org/plugins/wpscan/) to check for outdated or vulnerable plugins. If you wish to use this service to receive an alerts when your plugins need to be updated, you'll need to obtain an [API token](https://wpscan.com/pricing) from their website and configure your site to use your token.
```

Comment:
I've updated the language in [a2f5a6d](https://github.com/pantheon-systems/documentation/pull/7246/commits/a2f5a6dab7d0a27946838d8bba9e904ca1be7889), and am looking into your second comment.

Reply:
It doesn't appear that the `PANTHEON_WPVULNDB_API_TOKEN` ever populated the WPScan plugin. If a customer puts `WPSCAN_API_TOKEN` in their `wp-config.php`it will satisfy the plugin.